src/functionalTest/resources/features/F-001 POST Hearing/S-001.1.td.json

```diff
@@ -12,7 +12,7 @@
     "body": {
       "_extends_": "createHearingRequestFullData",
       "caseDetails": {
-        "caseRef": "${[scenarioContext][childContexts][CreateCase][testData][actualResponse][body][id]}"
+        "caseRef": "1512672408575038"
```

Comment:
are we sure we want to hard code the case ref? should be fine if we can guarantee we use test stub for all calls that are intended for the data store

Reply:
removed hard coding and Case ref is generated now